GAE/src/org/akvo/flow/xml/XmlOption.java

```diff
@@ -41,11 +41,40 @@
     @JacksonXmlProperty(localName = "text", isAttribute = false)
     private String text;
     @JacksonXmlElementWrapper(localName = "altText", useWrapping = false)
-    private XmlAltText[] altText;
+    private List<XmlAltText> altText;
 
     public XmlOption() {
     }
 
+    public XmlOption(QuestionOption o) {
+        setCode(o.getCode());
+        setText(o.getText());
+        setValue(o.getText());
+        //Translations
+        altText = new ArrayList<>();
+        for (Translation t: o.getTranslationMap().values()) {
+            altText.add(new XmlAltText(t));
+        }
+    }
+
+    /**
+     * @return a DTO object with relevant fields copied
+     */
+    public QuestionOptionDto toDto() {
+        QuestionOptionDto dto = new QuestionOptionDto();
+        //TODO translations
```

Comment:
Will it work ok without this? And if so then maybe create a separate follow up issue and remove the TODO.

Reply:
Translations not needed for reporting.